### PR TITLE
test: add e2e test for config volumes

### PR DESCRIPTION
**Pull request by jrangelramos**
# Changes

- :broom: Adding e2e test for func config volumes

### CODE REVIEW

**Review by zroubalik — COMMENTED** [review]
/lgtm

Thanks for this!

File: test/_e2e/cmd_config_volumes_test.go [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+	knFunc.TestShell.ShouldDumpOnSuccess = false
+	knFunc.commandSleepInterval = time.Millisecond * 1500
+
+	// On When...
+	project := FunctionTestProject{}
+	project.Runtime = "go"
+	project.Template = "volumes"
+	project.FunctionName = "test-config-volumes"
+	project.ProjectPath = filepath.Join(os.TempDir(), project.FunctionName)
+	project.RemoteRepository = "http://github.com/jrangelramos/test-templates.git"
```
[the remark this answers is not in the document]
Comment (lance) [reply test/_e2e/cmd_config_volumes_test.go:103]: > Needs to be changes when [boson-project/test-templates#4](https://github.com/boson-project/test-templates/pull/4) is accepted

Done

File: test/_e2e/cmd_config_volumes_test.go [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+	config, _ := k8s.GetClientConfig().ClientConfig()
+	clientset, _ := kubernetes.NewForConfig(config)
+	namespace, _, _ := k8s.GetClientConfig().Namespace()
+
+	_ = clientset.CoreV1().ConfigMaps(namespace).Delete(context.Background(), "test-cm-volume", metav1.DeleteOptions{})
+	_ = clientset.CoreV1().Secrets(namespace).Delete(context.Background(), "test-secret-volume", metav1.DeleteOptions{})
+
+}
+
+// ConfigVolumesAdd generate sa go function to test `func config volumes add` with user input
```
Comment (lance) [inline test/_e2e/cmd_config_volumes_test.go:73]:
```suggestion
// ConfigVolumesAdd generates a go function to test `func config volumes add` with user input
```
The tiniest of nits

**Review by lance — COMMENTED** [review]
/lgtm

**Pull request merged**